portfolio/src/main/webapp/script.js

```diff
@@ -54,3 +54,26 @@ window.onload = () => {
     .then(text => { document.getElementById("post1").innerHTML = text; 
   });
 }
+
+/** 
+ * Fetch messages from server and insert them on the home page.
+ */
+window.onload = () => {
```

Comment:
Similar to the previous PR, let's have a single window.onload for the file, and refactor the various fetches.